local/include/etc/confluent/docker/run

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
is this file any different than the version in cp-kafka?

Reply:
Yes. Here we are configuring default values as well